domain-rrd/rrd-shared/src/main/java/gov/va/vro/model/rrd/mas/MasAnnotation.java

```diff
@@ -16,9 +18,11 @@
 @Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class MasAnnotation {
+  // Maps to one of the values in the enum type @code{MasAnnotType}
   @JsonProperty("annot_type")
   private String annotType;
 
+  // Refers to the location of the extracted @code{annotVal} text within the document
   @JsonProperty("page_num")
```

Comment:
From what I have seen this can be multiple pages. Typically comma separated from what I remember. Not sure if it can be dash separated as well.